sdk/python/kfp/dsl/_container_op.py

```diff
@@ -871,6 +872,16 @@ def add_pod_label(self, name: str, value: str):
         self.pod_labels[name] = value
         return self
 
+    def set_retry_policy(self, policy: str):
+        """Sets the retry policy for a failed task. Possible values: Always, OnFailure, OnError.
+
+        Args:
+          policy: Retry policy name.
+        """
+
+        self.retry_policy = policy
```

Comment:
Shall we add validation if the policy is actually one of `Always, OnFailure, OnError`?

Reply:
+1

Reply:
Validation added.